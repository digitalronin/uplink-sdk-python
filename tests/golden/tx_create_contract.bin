00000000: 00 00 00 00 00 B2 0A 67  6C 6F 62 61 6C 20 69 6E  .......global in
00000010: 74 20 78 20 3D 20 30 20  3B 0A 0A 74 72 61 6E 73  t x = 0 ;..trans
00000020: 69 74 69 6F 6E 20 69 6E  69 74 69 61 6C 20 2D 3E  ition initial ->
00000030: 20 73 65 74 3B 0A 74 72  61 6E 73 69 74 69 6F 6E   set;.transition
00000040: 20 73 65 74 20 2D 3E 20  74 65 72 6D 69 6E 61 6C   set -> terminal
00000050: 3B 0A 0A 40 73 65 74 0A  65 6E 64 20 28 29 20 7B  ;..@set.end () {
00000060: 0A 20 20 74 65 72 6D 69  6E 61 74 65 28 22 4E 6F  .  terminate("No
00000070: 77 20 49 20 64 69 65 2E  22 29 3B 0A 7D 0A 0A 40  w I die.");.}..@
00000080: 69 6E 69 74 69 61 6C 0A  73 65 74 58 20 28 69 6E  initial.setX (in
00000090: 74 20 79 29 20 7B 0A 20  20 78 20 3D 20 34 32 3B  t y) {.  x = 42;
000000A0: 0A 20 20 74 72 61 6E 73  69 74 69 6F 6E 54 6F 28  .  transitionTo(
000000B0: 3A 73 65 74 29 3B 0A 7D                           :set);.}