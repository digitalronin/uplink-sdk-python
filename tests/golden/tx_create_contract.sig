r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 65334090145626808497726659100153416432834926697372486025123526118162194302386
