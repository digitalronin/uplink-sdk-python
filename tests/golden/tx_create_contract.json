{
    "timestamp": 1231006505, 
    "to": null, 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "CreateContract", 
            "contents": {
                "script": "", 
                "timestamp": 1231006505, 
                "address": "43WRxMNcnYgZFcE36iohqrXKQdajUdAxeSn9mzE1ZedB", 
                "owner": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE4xMDU5Nzc3OTE1MzUwODU2NTI1OTQ3NTU1MzI0MTg5MTk4NTU4MzQyNDEwMDk1MTgwNjE3MTg4MDM4MDg4NTY2OTkzOTMxMjc1MDcwMDQ=", 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
}